src/arch/arm/lib/debug/stack_iter.c

```diff
@@ -18,11 +18,11 @@
 #include "stack_iter.h"
 
 int stack_iter_next(stack_iter_t *f) {
-	f->fp = (void*) *((int*)f->fp - 3);
-	f->pc = (void*) *((int*)f->fp);
-	f->lr = (void*) *((int*)f->fp - 1);
+	f->fp = (void *) *((int *) (f->fp - 0xc));
+	f->pc = (void *) *((int *)f->fp);
+	f->lr = (void *) *((int *)f->fp - 0x4);
```

Comment:
Access 4 _words_ below `fp`, are parentheses required? `(f->fp - 0x4)`

Reply:
Yep. Added it.